libs/ledger-live-common/src/bridge/impl.ts

```diff
@@ -25,28 +25,41 @@ export const getCurrencyBridge = (currency: CryptoCurrency): CurrencyBridge => {
     currencyName: currency.name,
   });
 };
+
 export const getAccountBridge = (
   account: AccountLike,
   parentAccount?: Account | null,
 ): AccountBridge<any> => {
   const mainAccount = getMainAccount(account, parentAccount);
   const { currency } = mainAccount;
-  const { family } = currency;
-  const { type } = decodeAccountId(mainAccount.id);
   const supportedError = checkAccountSupported(mainAccount);
 
   if (supportedError) {
     throw supportedError;
   }
 
-  if (type === "mock") {
-    const mockBridge = mockBridges[currency.family];
-    if (mockBridge) return mockBridge.accountBridge;
+  try {
+    return getAccountBridgeByFamily(currency.family, mainAccount.id);
```

Comment:
We do not have a bridge<T> generic type anymore,  i understand it needs to be implemented differently, what is your vision on how we can support types in the future ?

Reply:
By generic, you mean using "typescript generic"? If so, it has never truly worked IMHO.
In the future I think it is good things to leverage Generics, but there are some works to do on it IMO.

Reply:
Yes this is what I meant, ok then